### ISSUE TITLE
Crash when try to set output path for scss compiler. Koala 2.0.2, Mac Os x 10.9.2

### ISSUE THREAD

**Issue by suchigg** [oklai/koala#190]
As explained in title, when i open the window for setting custom output path for compiler koala crash. The issue seems by node-webkit

**Comment by idmedia-kannengiesser** [comment]
I have the same problem, although downloaded the newest version...
is there any workaround?

**Comment by disalinas** [comment]
Is not fixed yet? I have the same problem.

**Comment by zaygraveyard** [comment]
Is this still an issue in the latest version?

**Comment by 13ogue** [comment]
yes it's still an issue, koala crashes several times a day and its not hidden in the system tray of win 10 machine that i am using, i can confirm that a 100%
